lib/tests/cleanup_before.rb

```diff
@@ -24,6 +24,20 @@ def run!(args:)
             end
           elsif OS.mac?
             delete_or_move Pathname.glob(HOMEBREW_CELLAR/"*")
+
+            frameworks_dir = Pathname("/Library/Frameworks")
+            %w[
+              Mono.framework
+              PluginManager.framework
+              Python.framework
+              R.framework
+              Xamarin.Android.framework
+              Xamarin.Mac.framework
+              Xamarin.iOS.framework
+            ].each do |framework|
+              path = frameworks_dir/framework
+              test "sudo", "mv", path, HOMEBREW_TEMP if path.exist?
```

Comment:
Can this use `delete_or_move` instead?

Reply:
Needs `sudo`, unfortunately. Would you rather we refactor `delete_or_move` to use `sudo` when requested?

Reply:
@carlocab Might make sense but: up to you, no strong feelings other way!

Reply:
Done!